doc/manual/src/language/advanced-attributes.md

```diff
@@ -329,3 +329,40 @@ Derivations can declare some infrequently used optional attributes.
     This is useful, for example, when generating self-contained filesystem images with
     their own embedded Nix store: hashes found inside such an image refer
     to the embedded store and not to the host's Nix store.
+
+  - [`__impure`]{#adv-attr-__impure}\
+    > **Warning**
+	  > This is an experimental feature.
+    >
+    > To enable it, add the following to [nix.conf](../command-ref/conf-file.md):
+    >
+    > ```
+  	> extra-experimental-features = impure-derivations
+	  > ```
+
+    If the **experimental** attribute `__impure` is set to `true`, then the
```

Comment:
```suggestion
    If the attribute `__impure` is set to `true`, then the
```

Attributes in themselves are not really experimental, and it already says above that this is an experimental feature.